pages/learn/index.page.tsx

```diff
@@ -7,7 +7,7 @@ import Card from '~/components/Card';
 import { DocsHelp } from '~/components/DocsHelp';
 
 export default function Welcome() {
-  const markdownFile = '_indexPage';
+  const fileRenderType = 'tsx';
```

Comment:
The more suitable fileRenderType would be `indexmd`. However, the markdown file lacks consistency in naming and has a custom name. Therefore, we should either rectify the file name and adhere to a standard convention or introduce an option to specify custom markdown file names for scenarios like these.

Reply:
@DarhkVoyd Which md file i should render in this case?